readme.md

```diff
@@ -1765,6 +1839,7 @@ You can even inspect and change the props of components, and see the results imm
 - [Write to stdout](examples/use-stdout/use-stdout.js) - Write to stdout bypassing main Ink output.
 - [Write to stderr](examples/use-stderr/use-stderr.js) - Write to stderr bypassing main Ink output.
 - [Static](examples/static/static.js) - Use `<Static>` to render permanent output.
+- [Router with Select Input](examples/router/select-input.js) - Change the url of the Router with a UI menu. Requires the additional packages `react-router` and [ink-select-input](https://github.com/vadimdemedes/ink-select-input).
```

Comment:
I think the description here could be improved. The `Requires the additional packages `react-router` and [ink-select-input](https://github.com/vadimdemedes/ink-select-input).` part is also not relevant and can be documented in the file itself if needed.